css/index.css

```diff
@@ -381,6 +381,77 @@
     padding-bottom: 150px;
   }
 
+
+
+#projects .container1{
+    width: 98%;
+    padding: 1%;
+    float:left;
+  }
+
+  #projects .main{
+    width: 98%;
+    margin-bottom: 1%;
+    height: auto;
+    padding: 1%;
+  }
+
+  #projects .main .row{
+    margin-left: 3%;
+  }
+
+#projects .heading{
+
+      font-size: 30px;
+      color: #5258a8;
+      text-transform: uppercase;
+      font-weight: bold;
+}
+
+ #projects .con{
+      border-radius: 0px !important;
+      background-color: white;
+      margin-top:22px;
+      margin-bottom: 22px;
+      overflow:hidden;
+      border:1px solid #7386D5;
+      box-sizing: border-box;
+      box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+      /* border-top-left-radius: 5%;
+      border-top-right-radius: 5%;  */
+
+    }
+#projects .main{
+
+      width: 98%;
+      margin-bottom: 1%;
+      height: auto;
+      /*padding: 1%;*/
+      font: black;
+      padding-bottom: 10%;
+    }
+#projects .project1{
+      /* border-radius: 8px; */
+      background-color:#7386D5 ;
+      color: #fafafa;
+      padding-left:1%;
+      padding-top:0.8%;
+      padding-bottom: 0.8%;
+      padding-left: 30px;
+      font-size:20px;
+    }
+#projects p{
+    color: black;
+  }
+#projects a{
+    color:blue;
+  }	
+  .projects {
+    padding-bottom: 150px;
+  }
+
```

Comment:
add something like, `.projects p{margin:10px;}` to increase the margin for the content